components/server/src/data/datamodel.json

```diff
@@ -4442,15 +4442,15 @@
                         "Robot Framework",
                         "Robot Framework Jenkins plugin",
                         "Snyk",
-                        "Generic",
+                        "Generic Security Warning",
```

Comment:
"Generic Security Warning" isn't a very clear name for the source type. How about "JSON file with security warnings"?